xbmc/cores/AudioEngine/Sinks/AESinkStarfish.cpp

```diff
@@ -13,9 +13,17 @@
 #include "utils/log.h"
 #include "xbmc/cores/AudioEngine/AESinkFactory.h"
 
-#include <chrono>
 #include <thread>
 
+namespace
+{
+constexpr auto STARFISH_AUDIO_BUFFERS = 8;
+constexpr auto AC3_SYNCFRAME_SIZE = 2560;
+} // namespace
+
+using namespace std::chrono;
```

Comment:
Is it preferable to put `using std::chrono` into the functions where it's actually used to limit the scope or just type out the types completely?

Reply:
I'm sorry if it wasn't clear but using the line `using namespace std::chrono_literals;` is ok as it makes it nice to be able to use the literals. Using `std::chrono` was what I was talking about here as we generally don't scope the std namespace.

I'm also fine if you want to use `using std::chrono` within functions where appropriate but it may be more annoying if some have it and some don't so I just write it out completely.